app/Nova/Resources/Coupon.php

```diff
@@ -32,20 +33,41 @@ final class Coupon extends Resource
         'is_distributed',
     ];
 
+    public static array $defaultSort = [
+        'is_distributed' => 'asc',
+    ];
+
+    public static function indexQuery(NovaRequest $request, $query): Builder
```

Comment:
Keep in mind that the proposed solution https://github.com/laravel/nova-issues/issues/156#issuecomment-415974405 adds this `indexQuery` method inside the custom `Resource`. By doing this it works on every Resource. But we deleted this custom `Resource` because it had no custom methods. Maybe it's worth bringing that file back?

Reply:
As of now we don't need a default sort in the other `Resources`, but it is indeed prudent to assume this might change in the future. I'll make the changes.